manifests/kilo-kubeadm-flannel-userspace.yaml

```diff
@@ -0,0 +1,176 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kilo
+  namespace: kube-system
+  labels:
+    app.kubernetes.io/name: kilo
+data:
+  cni-conf.json: |
```

Comment:
Do we need this with flannel mode? We don't have it in `manifests/kilo-kubeadm-flannel.yaml`

Reply:
yes, i need it

Reply:
Why is that exactly? Flannel should be providing CNI and all of the CNI binaries so Kilo should be able to be lazy

Reply:
oh, sorry,wo dont`t  need the configmap named kilo when user falannel

Reply:
Ah ok great! So we can simplify this manifest a bit

Reply:
done